meson_options.txt

```diff
@@ -0,0 +1,16 @@
+option('MFX_DECODER', type : 'boolean', value : true)
+option('USE_HEVC_DECODER', type : 'boolean', value : true)
+option('USE_VP8_DECODER', type : 'boolean', value : true)
+option('USE_VP9_DECODER', type : 'boolean', value : true)
```

Comment:
will do. compilation will be fine with latest version of MSS/MSDK though, it doesn't matter how old is the system. But we could switch off and log to the user by using the MFX API level in the build system to sanitize the options, what do you think?